app/uk/gov/hmrc/rasapi/connectors/DesConnector.scala

```diff
@@ -67,15 +71,24 @@ trait DesConnector extends ServicesConfig {
       MdcLoggingExecutionContext.fromLoggingDetails(rasHeaders))
 
     result.map(response => resolveResponse(response, userId, member.nino)).recover {
-      case ex: NotFoundException =>
+      case badRequestEx: BadRequestException =>
+        Logger.error("[DesConnector] [getResidencyStatus] Bad Request returned from des. The details sent were not valid.")
+        Right(ResidencyStatusFailure(error_BadRequest, "The pension scheme member's details do not conform to validation rules."))
```

Comment:
please mask this as InternalServerError , ideally this shouldn't happen if it does it will be the issue with the ras-api service and we can only provide response to the user as technical error